src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_onnx.py

```diff
@@ -0,0 +1,184 @@
+import inspect
+import warnings
+from typing import List, Optional, Union
+
+import torch
+
+from transformers import CLIPFeatureExtractor, CLIPTokenizer
+
+from ...onnx_utils import OnnxModel
+from ...pipeline_utils import DiffusionPipeline
+from ...schedulers import DDIMScheduler, LMSDiscreteScheduler, PNDMScheduler
+from . import StableDiffusionPipelineOutput
+
+
+class StableDiffusionOnnxPipeline(DiffusionPipeline):
+    vae_decoder: OnnxModel
+    text_encoder: OnnxModel
+    tokenizer: CLIPTokenizer
+    unet: OnnxModel
+    scheduler: Union[DDIMScheduler, PNDMScheduler, LMSDiscreteScheduler]
+    safety_checker: OnnxModel
+    feature_extractor: CLIPFeatureExtractor
+
+    def __init__(
+        self,
+        vae_decoder: OnnxModel,
+        text_encoder: OnnxModel,
+        tokenizer: CLIPTokenizer,
+        unet: OnnxModel,
+        scheduler: Union[DDIMScheduler, PNDMScheduler, LMSDiscreteScheduler],
+        safety_checker: OnnxModel,
+        feature_extractor: CLIPFeatureExtractor,
+    ):
```

Comment:
The main difference it that the models are replaced with `diffusers.onnx_utils.OnnxModel`